tests/test_colorbars_reference.tex

```diff
@@ -1,69 +1,3 @@
 \begin{tikzpicture}
 
-\begin{groupplot}[group style={group size=1 by 3}]
-\nextgroupplot[
-tick align=outside,
-tick pos=left,
-x grid style={white!69.019608!black},
-xlabel={Some Units},
-xmin=-5, xmax=10,
-xtick style={color=black},
-ymin=-5, ymax=10
-]
-\path [draw=white, fill=white, line width=0.004pt]
-(axis cs:-5,-5)
---(axis cs:-4.9414062,-5)
---(axis cs:9.9414062,-5)
---(axis cs:10,-5)
---(axis cs:10,10)
---(axis cs:9.9414062,10)
---(axis cs:-4.9414062,10)
---(axis cs:-5,10)
---(axis cs:-5,10)
---cycle;
-\addplot graphics [includegraphics cmd=\pgfimage,xmin=-5, xmax=10, ymin=-5, ymax=10] {tmp-000.png};
-
-\nextgroupplot[
-tick align=outside,
-tick pos=left,
-x grid style={white!69.019608!black},
-xlabel={Discrete intervals, some other units},
-xmin=0.65, xmax=8.35,
-xtick style={color=black},
-ymin=1, ymax=8
-]
-\path [draw=white, fill=white, line width=0.004pt]
-(axis cs:0.65,4.5)
---(axis cs:1,1)
---(axis cs:8,1)
---(axis cs:8.35,4.5)
---(axis cs:8.35,4.5)
---(axis cs:8,8)
---(axis cs:1,8)
---(axis cs:0.65,4.5)
---cycle;
-\addplot graphics [includegraphics cmd=\pgfimage,xmin=0.65, xmax=8.35, ymin=1, ymax=8] {tmp-001.png};
-
-\nextgroupplot[
-tick align=outside,
-tick pos=left,
-x grid style={white!69.019608!black},
-xlabel={Custom extension lengths, some other units},
-xmin=-1.5, xmax=1.5,
-xtick style={color=black},
-ymin=-1, ymax=1
-]
-\path [draw=white, fill=white, line width=0.004pt]
-(axis cs:-1.5,0)
---(axis cs:-1,-1)
---(axis cs:1,-1)
---(axis cs:1.5,0)
---(axis cs:1.5,0)
---(axis cs:1,1)
---(axis cs:-1,1)
---(axis cs:-1.5,0)
---cycle;
-\addplot graphics [includegraphics cmd=\pgfimage,xmin=-1.5, xmax=1.5, ymin=-1, ymax=1] {tmp-002.png};
-\end{groupplot}
-
 \end{tikzpicture}
```

Comment:
This looks like a regression, see also https://github.com/nschloe/tikzplotlib/issues/606
What happened here?